magento/graphql-ce
# Small image URL added to the product data

### Description
This PR adds an extra field to the `ProductInterface` with image URL. We already have the `small_image` field with information about the image path. However, the image path information is not very useful for PWA applications that build a list of products and require rather URLs than some data that needs to be parsed.

### Fixed Issues (if relevant)
1. https://github.com/magento/graphql-ce/issues/88 : 'small_image' in 'ProductInterface' does not have enough info

### Manual testing scenarios
1. Create a new product with/without an image
2. Make the following request:

```
{
  products (search: "withimages") {
    items {
      small_image_url
    }
  }
}
```
3. You should see the small image URL in the response:

```
{
  "data": {
    "products": {
      "items": [
        {
          "small_image_url": "http://docker.graphql.loc:8087/media/catalog/product/cache/0c40cc075c35680261029fa6d255feee/1/6/16262-11.jpg"
        }
      ]
    }
  }
}
```
4. If the requested product does not have an image, a placeholder URL should be returned
